.gitignore

```diff
@@ -15,3 +15,4 @@
 /src/main/generated/resources/.cache/
 /.metadata
 /logs/
+java_pid29320.hprof
```

Comment:
Do you know what this is?

Reply:
it's a file that showed up when i started the 1.21 port. figured i'd leave the gitignore change it in just in case

Reply:
I gave it a search, and it is likely a log file (specifically a heap dump) from a java crash (from JVM, not from the program), _probably_ caused by an out of memory error. I would guess that the number is just an arbitrary identifier for that specific crash. You should be fine to get rid of it, or move it out of the project folder if you want to give it a closer look later.